packages/cacti-ledger-browser-react/src/main/typescript/components/AppShell/Home.tsx

```diff
@@ -1,14 +1,18 @@
 import styles from "./Home.module.css";
-import logo from "../../../../../../images/HL_Cacti_Logo_Color.png";
+import logo from "../../../../../../../images/HL_Cacti_Logo_Color.png";
+import StatusPage from "../../apps/cacti/pages/status-page";
+
 const Home = () => {
   return (
     <div className={styles.home}>
       <p className={styles["home-scale"]}>
         Select ledger from the dropdown menu
       </p>
-      <span className={styles["home-icon"]}>
+
+      <StatusPage></StatusPage>
+      {/* <span className={styles["home-icon"]}>
         <img className={styles["home-scale"]} src={logo} alt="Logo" />
-      </span>
+      </span> */}
```

Comment:
@rwat17 can you remove this comment block?

Reply:
@izuru0 Done